crates/store/re_types/definitions/rerun/blueprint/views/map.fbs

```diff
@@ -0,0 +1,12 @@
+namespace rerun.blueprint.views;
+
+/// A map view.
```

Comment:
a bit more description of a what map view is capable of would be nice. Keep in mind that this is the main entry point for "these are the view types the viewer has and that's what they do".